### PR TITLE
Add German date format

### PR DESCRIPTION
**Pull request by wildi1**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by benbrich — APPROVED** [review]
Just tested and it works fine! How about merging this one?

**Comment by benbrich** [comment]
@pedrobaeza is there a reason, why not merging this one? Any Problems with the code in other countries?

**Comment by pedrobaeza** [comment]
I'm not anymore OCA's community manager and I'm not monitoring PRs. This should corresponds to the PSC (as in the past, but I served as wildcard). I can do it one last time, but now this PR is contaminated with 4 commits more than when I approved it.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remove extra commits, that also performs other things.

**Comment by benbrich** [comment]
Hi and thanks, I understand! Wanted to address @StefanRijnhart first, but thought you are on the PSC and on the topic. Couldn't find a maintainer in the readme. Will remember next time...

**Comment by benbrich** [comment]
@wildi1 would be great if you could fix this soon! :-)

File: account_bank_statement_import_paypal/wizards/account_bank_statement_import_paypal.py
Comment (StefanRijnhart) [inline account_bank_statement_import_paypal/wizards/account_bank_statement_import_paypal.py:159]: I'm slightly puzzled by this line. It looks like you encountered a statement line with the currency amount having the wrong sign. Can you clarify this a bit, or even better, add a test for this situation?

**Comment by wildi1** [comment]
@pedrobaeza The first PR was made in Aug 19. I thought it will be merged soon after you approved it and so I forgot that it is not merged yet. Therfore my other commits have contaminated this PR.

@StefanRijnhart This schould be part of an other Pull Request I already made. I have split them now in two separate PRs.

Here is the new PR for this request.
#256

**Comment by wildi1** [comment]
@StefanRijnhart Here is the other PR and the explenation of the behaviour with the wrong sign.
 #258

**Comment by wildi1** [comment]
@benbrich I have fixed it soon :) When you are using Paypal with different currencies you should also have a look at #258

**Pull request closed without merging**
